### PR TITLE
[WARP] Added WARP as Local Proxy instructions

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

**Review by kkrum — APPROVED** [review]

File: products/warp-client/src/content/overview.md [hunk 1]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
 
 ### WARP Modes
 
-* **1.1.1.1** is the fastest and most private DNS directory you can configure for all your devices. It also allows you to add security to your internet connection, because it gives you the option to send your DNS requests over DNS-over-HTTPS or DNS-over-TLS. This will ensure nobody will be able to eavesdrop on you.
+### 1.1.1.1
+
+**1.1.1.1** is the fastest and most private DNS directory you can configure for all your devices. It also allows you to add security to your internet connection, because it gives you the option to send your DNS requests over DNS-over-HTTPS or DNS-over-TLS. This will ensure nobody will be able to eavesdrop on you.
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md:12]: change to `Internet`

File: products/warp-client/src/content/overview.md [hunk 2]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
 
  Read more about 1.1.1.1 on our blog post [Announcing 1.1.1.1: the fastest, privacy-first consumer DNS service](https://blog.cloudflare.com/announcing-1111/) and [Introducing 1.1.1.1 for Families](https://blog.cloudflare.com/introducing-1-1-1-1-for-families/)
 
-* **WARP.** The WARP application uses [BoringTun](https://blog.cloudflare.com/boringtun-userspace-wireguard-rust/) to encrypt all of the traffic from your device and send it directly to Cloudflare’s edge, ensuring that no one in between is snooping on what you're doing. If the site you are visiting is already a Cloudflare customer, the content is immediately sent down to your device. If they aren't on our network, we use our global network of data centers to devise the shortest path to whomever you are talking to.
+### 1.1.1.1 with WARP
+
+<Aside>
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: change to `<Aside type="note">`

File: products/warp-client/src/content/overview.md [hunk 3]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
+ 
+ **WARP** does not provide anonymity. It is not designed to prevent servers you communicate with from identifying you, or to allow you to pretend to be accessing the Internet in a different country than you are currently in.
+
+</Aside>
+
+**WARP.** The WARP application uses [BoringTun](https://blog.cloudflare.com/boringtun-userspace-wireguard-rust/) to encrypt all of the traffic from your device and send it directly to Cloudflare’s edge, ensuring that no one in between is snooping on what you're doing. If the site you are visiting is already a Cloudflare customer, the content is immediately sent down to your device. If they aren't on our network, we use our global network of data centers to devise the shortest path to whomever you are talking to.
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: `you are doing`

`If they are not on your network`

File: products/warp-client/src/content/overview.md [hunk 4]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
 
  Read more about WARP in our blog post [Introducing WARP: Fixing Mobile Internet Performance and Security](https://blog.cloudflare.com/1111-warp-better-vpn/)
 
-  <Aside>
+### WARP via Local Proxy
+
+<Aside>
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md:30]: add `<Aside type="note">`

File: products/warp-client/src/content/overview.md [hunk 5]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
  
- **WARP** does not provide anonymity. It is not designed to prevent servers you communicate with from identifying you, or to allow you to pretend to be accessing the Internet in a different country than you are currently in.
+ Currently available on desktop clients only.
+
+</Aside>
+
+When WARP is configured as a local proxy, only the applications that you configure to use the proxy (HTTPS or SOCKS5) will have their traffic sent through WARP. This allows you to pick and choose which traffic is encrypted (for instance, your web browser or a specific app), and everything else will be left open over the Internet. To enav
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: Missing content at the end of the sentence

File: products/warp-client/src/content/overview.md [hunk 6]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
+
+Because this feature restricts WARP to just applications configured to use the local proxy, leaving all other traffic unencrypted over the Internet by default, we’ve hidden it in the advanced menu. To turn it on:
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: `we have hidden it`

File: products/warp-client/src/content/overview.md [hunk 7]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
+
+1. Navigate to Preferences > Advanced and click **Configure Proxy**.
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: `**Preferences** > **Advanced**`

File: products/warp-client/src/content/overview.md [hunk 8]
```diff
@@ -7,20 +7,42 @@ Cloudflare WARP supports the following services depending on how you configure i
...
+2. On the dialog that opens, check the box and configure the port you want to listen on.
+3. This will enable the **WARP via Local Proxy** option in the WARP Settings menu.
```
Comment (marciocloudflare) [inline products/warp-client/src/content/overview.md]: Change numbers to `1.`

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**